cpp/dolfinx/fem/assemble_matrix_impl.h

```diff
@@ -201,12 +242,12 @@ void assemble_exterior_facets(
     kernel(Ae.data(), coeffs.data() + index / 2 * cstride, constants.data(),
            coordinate_dofs.data(), &local_facet, nullptr);
 
-    P0(_Ae, cell_info, cell, ndim1);
-    P1T(_Ae, cell_info, cell, ndim0);
+    P0(_Ae, cell_info1, cell1, ndim1);
```

Comment:
Are you sure things are the correct way around here?

Reply:
Whoops, these aren't right. Well spotted!

Reply:
Not that I'm 100% sure either . . .

Reply:
It was wrong before, the DOF transformation for the test element should be applied using the test mesh cell info, indexed using the test mesh cell